sdk/cosmos/azure-cosmos/azure/cosmos/aio/_database.py

```diff
@@ -363,11 +459,14 @@ async def create_container_if_not_exists(
                 DeprecationWarning)
         try:
             container_proxy = self.get_container_client(id)
-            await container_proxy.read(
+            headers = await container_proxy.read(
```

Comment:
```suggestion
            response = await container_proxy.read(
```
Not that it really matters, but technically this isn't the headers....